packages/core/src/odata/v4/batch-request-builder.ts

```diff
@@ -30,6 +31,13 @@ export class ODataBatchRequestBuilderV4 extends BatchRequestBuilder {
     options?: DestinationOptions
   ): Promise<BatchResponse[]> {
     return this.build(destination, options)
+      .then(request => {
```

Comment:
I would expect this to be in the build method, which returns the request right?

Reply:
I think for the other request builders it is also done in the build

Reply:
Yeah, I considered moving that, but that means that I have to also copy the overrides of build, I think. but yeah...